src/nplinker/metabolomics/GNPSSpectrumLoader.py

```diff
@@ -0,0 +1,47 @@
+from nplinker.annotations import GNPS_DATA_COLUMNS
+from nplinker.annotations import GNPS_KEY
+from nplinker.annotations import create_gnps_annotation
+from nplinker.logconfig import LogConfig
+from nplinker.metabolomics.ISpectrumLoader import ISpectrumLoader
+from nplinker.metabolomics.spectrum import Spectrum
+from nplinker.parsers.mgf import LoadMGF
+
+
+logger = LogConfig.getLogger(__file__)
+
+class GNPSSpectrumLoader(ISpectrumLoader):
+
+    def __init__(self, filename):
+        ms1, ms2, metadata = LoadMGF(name_field='scans').load_spectra([filename])
+        logger.info('%d molecules parsed from MGF file', len(ms1))
+        self._spectra = mols_to_spectra(ms2, metadata)
+    
+    def spectra(self):
+        return self._spectra
+    
+
+def mols_to_spectra(ms2, metadata):
```

Comment:
- turn this function to a method of class `GNPSSpectrumLoader`?
- add docstring and typing hint for it? 
- the `mols` in the name is confusing to me, rename the function to e.g. `parse_*`, `load_*`?

Reply:
mols stands for molecules?

Reply:
It also doesn't use any information from the class - this is currently a mere wrapper, so the LoadMGF class also isn't a member of this class at the moment. Should it all be moved in here for now to make it clear that this is all code related to loading spectra?

Reply:
No need to move LoadMGF class at the moment.

The `mols_to_spectra(ms2, metadata)` could be changed to a static method `parse_mgf(filename) -> Spectra`.